examples/exceptions_asserts.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+* This example shows how to make use of the builtin assert and exception system.
+*/
+
+#include <xtypes/xtypes.hpp>
+#include <iostream>
+
+// Auxiliar define to manage both assert (debug only) and exceptions (both)
+#if defined(XTYPES_EXCEPTIONS)
```

Comment:
Remove it and include utils :D

Reply:
It's not the same define here. It is full of comments and *couts* to show information ;).